.circleci/config.yml

```diff
@@ -231,7 +231,7 @@ jobs:
                   path: ~/transformers/test_preparation.txt
             - run: |
                   if [ -f test_list.txt ]; then
-                    python -m pytest -n 3 --dist=loadfile -s --make-reports=tests_torch $(cat test_list.txt) | tee tests_output.txt
+                    python -m pytest --timeout 300 -n 3 --dist=loadfile -s --make-reports=tests_torch $(cat test_list.txt) | tee tests_output.txt
```

Comment:
Not sure we can afford such a timeout for the basic tests.

Reply:
Yes, you are absolutely right. It was a temporary addition because I had the `Too long with no output (exceeded 10m0s): context deadline exceeded` error in CircleCI and this technique could let me know which test was long. But maybe it was just bad luck with some internet connection for example

With the new parameters, only `tests/test_tokenization_fnet.py::FNetTokenizationTest::test_save_pretrained` timed out when it was converting the slow version to the fast version.

Reply:
Good new: this seems to be fixed with my last merge of the master branch!